packages/prediction-app/src/features/evaluation-compare/EvaluationCompare.tsx

```diff
@@ -0,0 +1,218 @@
+import {
+    ComparionPlotWrapper,
+    ComparisonPlotList,
+    SplitPeriodSelector,
+} from '@dhis2-chap/chap-lib'
+import {
+    EvaluationCompatibleSelector,
+    EvaluationSelectorBase,
+} from '../select-evaluation/EvaluationSelector'
+import React, { useEffect, useMemo } from 'react'
+import css from './EvaluationCompare.module.css'
+import {
+    Help,
+    IconArrowLeft16,
+    IconArrowRight16,
+    IconVisualizationLine24,
+    IconVisualizationLineMulti24,
+    MultiSelect,
+    MultiSelectOption,
+} from '@dhis2/ui'
+import i18n from '@dhis2/d2-i18n'
+import {
+    useSelectedEvaluationsController,
+    useSelectedOrgUnits,
+    useSelectedSplitPoint,
+} from './useSelectEvaluations'
+import { usePlotDataForEvaluations } from '../../hooks/usePlotDataForEvaluations'
+import { useEvaluationOverlap } from '../../hooks/useEvaluationOverlap'
+import { useOrgUnitsById } from '../../hooks/useOrgUnitsById'
+import PageHeader from '../common-features/PageHeader/PageHeader'
+import OrganisationUnitMultiSelect from '../../components/OrganisationUnitsSelect/OrganisationUnitMultiSelect'
+
+const MAX_SELECTED_ORG_UNITS = 10
+
+export const EvaluationCompare = () => {
+    const {
+        query: evaluationsQuery,
+        baseEvaluation,
+        comparisonEvaluation,
+        setBaseEvaluation,
+        setComparisonEvaluation,
+    } = useSelectedEvaluationsController()
+
+    const selectedEvaluations = [baseEvaluation, comparisonEvaluation].filter(
+        (e) => !!e
+    )
+    const [selectedOrgUnits, setSelectedOrgUnits] = useSelectedOrgUnits()
+    const [selectedSplitPoint, setSelectedSplitPoint] = useSelectedSplitPoint()
```

Comment:
I realize the naming here (and in regards to evaluation vs backtest) is inconsistent. I will fix that in a follow up PR, to avoid conflicts, since https://github.com/dhis2-chap/chap-frontend-monorepo/pull/50 is branched off of this.